spec.md

```diff
@@ -991,6 +1149,18 @@ ports:
     mode: host
 ```
 
+
+### pull_policy
+
+`pull_policy` defines the decisions Compose implementations will make when it starts to pull images. Possible values are:
+
+* `always`: Compose implementations SHOULD always pull the image from the registry.
+* `never`: Compose implementations SHOULD NOT pull the image from a registry and SHOULD rely on the platform cached image. If there is no cached image, a failure MUST be reported.
+* `if_not_present`: Compose implementations SHOULD pull the image only if it's not available in the platform cache.This SHOULD be the default option for Compose implementations without build support.
```

Comment:
This is the equivalent of `--pull=missing` (https://github.com/docker/cli/pull/1498)? Was this how it was named in the v2 format, or renamed from what we had previously?

Reply:
I guess it was isnpired by Kubernetes `IfNotPresent` pull policy

Reply:
note: this one was introduced previously, only moved to use alphabetic ordering on attributes